specification/network/resource-manager/Microsoft.Network/stable/2022-01-01/firewallPolicy.json

```diff
@@ -1587,6 +1587,10 @@
           "items": {
             "type": "string"
           }
+        },
+        "autoLearnPrivateRanges": {
```

Comment:
>autoLearnPrivateRanges [](http://example.com/codeflow?start=9&length=22)

Consider using string enum, instead of boolean (only true/false) something more like:

privateRangeHandling: { Fixed, LearnSnat }, or similar.

Reply:
Fixed